griptape/drivers/image_generation_model/bedrock_stable_diffusion_image_generation_model_driver.py

```diff
@@ -115,7 +115,7 @@ def _request_parameters(
         }
 
         if image:
-            request["init_image"] = image.base64
+            request["init_image"] = image.value
```

Comment:
This change passes raw bytes to the image model APIs that expect base64-encoded images.